### PR TITLE
Image Hovering

### PR DESCRIPTION
**Pull request by Alee-star**
Image Hover Component

### CODE REVIEW

File: src/components/Banner/ImageCard.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  list-style: none;
+}
+
+.icon-image {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 40px;
+  height: 40px;
+  border: 0;
```
Comment (vonnue-nidheesh) [inline src/components/Banner/ImageCard.css]: Should we need to specify `border:0` here?

File: src/components/Banner/ImageCard.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
+.image-card {
+  position: relative;
+  width: 100%;
+  overflow: hidden;
+  height: 100%;
+  border-radius: 0;
```
Comment (vonnue-nidheesh) [inline src/components/Banner/ImageCard.css]: Should we need to specify `border-radius: 0;` here?

File: src/components/Banner/ImageCard.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        <img src={photo.url} alt={photo.title} className="card-image" />
+      </div>
+      {hovered && (
+        <div className="hover-part">
+          <ul className="hover-top">
+            <li className="hover-icon">
+              <Button
+                className="icon-image"
+                label={<img src="/assets/copy.svg" alt="copy" />}
+              ></Button>
```
Comment (vonnue-nidheesh) [inline src/components/Banner/ImageCard.jsx]: Self close

File: src/components/Banner/ImageCard.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      <div className="card">
+        <img src={photo.url} alt={photo.title} className="card-image" />
+      </div>
+      {hovered && (
+        <div className="hover-part">
+          <ul className="hover-top">
+            <li className="hover-icon">
+              <Button
+                className="icon-image"
+                label={<img src="/assets/copy.svg" alt="copy" />}
```
Comment (vonnue-nidheesh) [inline src/components/Banner/ImageCard.jsx]: alt="Copy icon"

An alt(alternative text) must be in human-readable form.

File: src/components/Banner/ImageCard.jsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+              ></Button>
+            </li>
+            <li className="hover-icon">
+              <Button
+                className="icon-image"
+                label={<img src="/assets/like.svg" alt="like" />}
+              ></Button>
```
Comment (vonnue-nidheesh) [inline src/components/Banner/ImageCard.jsx]: Same here

File: src/components/Banner/ImageCard.jsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+            <li className="hover-icon">
+              <Button
+                className="icon-image"
+                label={<img src="/assets/copy.svg" alt="copy" />}
+              ></Button>
+            </li>
+            <li className="hover-icon">
+              <Button
+                className="icon-image"
+                label={<img src="/assets/like.svg" alt="like" />}
```
Comment (vonnue-nidheesh) [inline src/components/Banner/ImageCard.jsx]: `alt="Like icon"`

File: src/components/Banner/ImageCard.jsx [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+              ></Button>
+            </li>
+          </ul>
+          <div className="hover-bottom">
+            <div className="download-part">
+              <Button
+                className="download-button"
+                label={<img src="/assets/download.svg" alt="download" />}
+              ></Button>
```
Comment (vonnue-nidheesh) [inline src/components/Banner/ImageCard.jsx]: Same here

File: src/components/Banner/ImageCard.jsx [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+            </li>
+            <li className="hover-icon">
+              <Button
+                className="icon-image"
+                label={<img src="/assets/like.svg" alt="like" />}
+              ></Button>
+            </li>
+          </ul>
+          <div className="hover-bottom">
+            <div className="download-part">
```
Comment (vonnue-nidheesh) [inline src/components/Banner/ImageCard.jsx:38]: I think we missed the image title here

**Review by vonnue-nidheesh — APPROVED** [review]

File: src/components/Banner/ImageCard.css (hunk 1 above)
Comment (Alee-star) [reply src/components/Banner/ImageCard.css]: other wise a black border is appearing

Comment (vonnue-nidheesh) [reply src/components/Banner/ImageCard.css]: Then keep it like this. We can check this later.

Comment (Alee-star) [reply src/components/Banner/ImageCard.css]: ok

File: src/components/Banner/ImageCard.css [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+  align-items: center;
+  max-width: 100%;
+}
+
+.hover-title {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  text-align: left;
+  left: 0;
```
Comment (vonnue-nidheesh) [inline src/components/Banner/ImageCard.css:65]: Should we need this left property? If not remove it in the next pr.

**Review by vonnue-nidheesh — APPROVED** [review]

**Pull request merged**